### DISCUSSION TITLE
Keybinding to `ga` to switch to previous buffer is missing.

### DISCUSSION THREAD

**Discussion by leira** [71/dance#231] [Q&A] [answered]
I like the `ga` keybinding in Kakoune. I use it a lot to switch between 2 files. There is no `a` in the `g` menu for Dance. I try to recreate such keybinding in vscode, but `workbench.action.previousEditor` continuously goes back in the previous list, rather than switching between the 2 recent files. Is it because of some limitation of vscode, so `ga` is missed in Dance?

**Comment by losnappas** [comment]
I'm personally using `"workbench.action.quickOpenPreviousRecentlyUsedEditorInGroup",` for `g a`, works really nice. Although takes a bit to accommodate for `g a <enter>` rather than just `g a`, but it's worth it.

**Comment by leira** [comment, marked as answer]
@losnappas, thanks for pointing me to the right direction! I just solved this issue with:

```json
  {
    "key": "g a",
    "command": "dance.run",
    "args": {
      "commands": [
        "workbench.action.quickOpenPreviousRecentlyUsedEditorInGroup",
        "list.select",
      ],
    }
  },
```

This will remove the <enter> needed in the end.

**Reply by 71** [reply]
Great tip! I didn't implement `g a` as I didn't want to maintain a history of open editors (though it kinda already exists internally, but that's another issue), but I didn't know about these commands. This menu item will be included in the next release of Dance (with `workbench.action.openPreviousRecentlyUsedEditorInGroup`, which doesn't require `list.select`): c16096ce1f73f4d817fb7091f3985b129d61b42b.

**Reply by Shatur** [reply]
@71 can we also have `gn` and `gp` like in helix?
VSCode have `workbench.action.nextEditor` and `workbench.action.previousEditor`.

**Reply by 71** [reply]
@Shatur Nice suggestion, thanks! Added in 64bf13e5fab09961952857beae3a2c09f600bff2.
